IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectiveness.mo

```diff
@@ -108,8 +107,10 @@ and <code>QMax_flow &gt; 0</code>.
 </html>", revisions="<html>
 <ul>
 <li>
-April 26, 2023, by Michael Wetter:<br/>
+April 27, 2023, by Michael Wetter:<br/>
 Set nominal and min attributes for capacity flow rates.<br/>
+Moved assignments from <code>initial equation</code> to parameter declaration.
+This avoids a warning in Dymola 2023x about non-literal nominal values.<br/>
```

Comment:
This also holds for Dymola 2022x. We could also just mention Dymola in general here? Or do you prefer to be as specific as possible in case these warnings would show up in a future release of Dymola?

Reply:
I prefer to add the version with which it was encountered in case someone needs to reproduce it.

Reply:
Making the statements in the `initial equation` section helps readability if the assignments are long. The two versions should be equivalent, but in this case, Dymola seem to treat them slightly different. I was surprised that this change removed the warnings.